src/flows/recommendation_api/user_impressions_flow.py

```diff
@@ -1,49 +1,84 @@
+  JOIN ANALYTICS.DBT.CONTENT AS c
+    ON c.CONTENT_ID = a.CONTENT_ID
+  WHERE CURRENT_DATE - DATE(a.TIME_ADDED) <= 1  
+  GROUP BY 1, 2, 3
+  )
+  
+SELECT
+    u.USER_ID,
+    p.HAPPENED_AT AS UPDATED_AT,
+    LISTAGG(DISTINCT p.RESOLVED_ID, ',') AS RESOLVED_IDS
+FROM prep as p
+JOIN ANALYTICS.DBT.USERS as u
+  ON p.HASHED_USER_ID = u.HASHED_USER_ID
+WHERE ((p.max_impression_age > %(MAX_AGE)s) OR (p.total_imprs > %(MAX_IMPRS)s))
+  AND CURRENT_DATE = p.HAPPEEND_AT
+GROUP BY 1, 2
+"""
 
 
 @task
-def transform_user_impressions_df(df: pd.DataFrame):
+def transform_user_impressions_df(df: pd.DataFrame) -> pd.DataFrame:
     df = df.rename(columns={"USER_ID": "user_id",
                             "RESOLVED_IDS": "resolved_ids",
-                            "LAST_UPDATED_AT": "updated_at"}).astype({"user_id": int})
+                            "UPDATED_AT": "updated_at"}).astype({"user_id": int})
+    df["resolved_ids"] = df.resolved_ids.apply(lambda x: "["+x+"]")
     df["updated_at"] = df.updated_at.apply(lambda x: x.strftime("%Y-%m-%dT%H:%M:%SZ"))
     return df
 
 
 @task
-def load_featue_group(df: pd.DataFrame, feature_group_name):
+def load_feature_group(df: pd.DataFrame, feature_group_name):
     boto_session = boto3.Session()
     feature_store_session = Session(boto_session=boto_session,
                                     sagemaker_client=boto_session.client(service_name='sagemaker'),
                                     sagemaker_featurestore_runtime_client=boto_session.client(service_name='sagemaker-featurestore-runtime'))
     feature_group = FeatureGroup(name=feature_group_name, sagemaker_session=feature_store_session)
     feature_group.ingest(df, max_workers=4, max_processes=4, wait=True)
 
+# Schedule to run every hour
+if config.ENVIRONMENT == config.ENV_PROD:
+    schedule = IntervalSchedule(interval=datetime.timedelta(days=1))
+else:
+    schedule = None
 
-with Flow(FLOW_NAME) as flow:
+with Flow(FLOW_NAME, schedule=schedule) as flow:
     feature_group = Parameter("feature group", default="user-impressions")
-    snowflake_bucket = "pocket-data-learning"
-    snowflake_prefix = "analytics-modeled-data/parquet/user_impressions"
-    list_files_task = S3List()
-    parquet_files = list_files_task(bucket=snowflake_bucket, prefix=snowflake_prefix)
-    dfs = extract_parquet_as_df.map(parquet_file=parquet_files, bucket=unmapped(snowflake_bucket))
-    xdfs = transform_user_impressions_df.map(dfs)
-    load_featue_group.map(df=xdfs, feature_group_name=unmapped(feature_group))
+    max_impr_age = Parameter("max impression age", default=6)
+    max_impr_count = Parameter("max impression count", default=12)
+
+    snowflake_result = PocketSnowflakeQuery()(
+        query=BASE_QUERY,
+        data={"MAX_AGE": max_impr_age, "MAX_IMPRS": max_impr_count}
+    )
+
+    xformed_df = transform_user_impressions_df(snowflake_result)
+    load_feature_group(df=xformed_df, feature_group_name=feature_group)
```

Comment:
```suggestion
    transformed_df = transform_user_impressions_df(snowflake_result)
    load_feature_group(df=transformed_df, feature_group_name=feature_group)
```
I got what this meant, but I usually like to write names out, unless the shortened name is used frequently, to lower the barrier for other devs.